server/routes/profile.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  newUser,
+  editProfile,
+  getProfile,
+  findSitters,
+} = require("../controllers/profile");
+
+router.route("/new").post(protect, newUser);
+router.route("/editprofile/:id").patch(protect, editProfile);

```

Comment:
if you have multiple words in the route, use hyphenated case :)
For Eg: `edit-profile`